File: report-generators/src/main/kotlin/org/jb/cce/EvaluationInfo.kt
package org.jb.cce

import org.jb.cce.actions.CompletionStrategy
import org.jb.cce.metrics.MetricInfo

data class EvaluationInfo (val evaluationType: String, val strategy: CompletionStrategy,
                           val filesInfo: Map<String, FileEvaluationInfo>, val metrics: List<MetricInfo>)

File: vscode-plugin/src/evaluationInfo.ts
import { FileSessions } from "./session";

export class SessionsEvaluationInfo {
    constructor(public sessions: FileSessions[], public info: EvaluationInfo) {}
}

export class EvaluationInfo {
    constructor(public evaluationType: string, public strategy: CompletionStrategy) {}
}

export class CompletionStrategy {
}